src/main/scala/cromwell/binding/types/WdlArrayType.scala

```diff
@@ -1,9 +1,10 @@
 package cromwell.binding.types
 
-import cromwell.binding.values.{WdlValue, WdlArray, WdlFile, WdlString}
+import cromwell.binding.values.{WdlArray, WdlFile, WdlString, WdlValue}
 import cromwell.util.TryUtil
 import spray.json.JsArray
-import scala.util.{Success, Failure}
+
+import scala.util.{Failure, Success}
```

Comment:
Yours seems to be set on "alphabetical" which would be both deterministic and sensible.  What are the rest of us doing?